src/shared/codelens/localLambdaRunner.ts

```diff
@@ -754,3 +731,19 @@ function getAttachDebuggerMaxRetryLimit(
         defaultValue
     )!
 }
+
+export function shouldAppendRelativePathToFunctionHandler(runtime: string): boolean {
+    // getFamily will throw an error if the runtime doesn't exist
+    switch (getFamily(runtime)) {
+        case SamLambdaRuntimeFamily.NodeJS:
+        case SamLambdaRuntimeFamily.Python:
+        case SamLambdaRuntimeFamily.Ruby:
+            return true
+        case SamLambdaRuntimeFamily.DotNet:
+        case SamLambdaRuntimeFamily.Java:
+        case SamLambdaRuntimeFamily.Go:
+        // if the runtime exists but for some reason we forgot to cover it here
+        default:
+            return false
```

Comment:
IMO if we forgot to cover the runtime here, we should throw instead of swallowing the error. Otherwise, we may forget to update this function when adding support for a new runtime.

Reply:
✔️